src/sentry/issues/issue_occurrence.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import hashlib
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Any, Mapping, Optional, Sequence
+
+from dateutil.parser import parse as parse_date
+from django.utils.timezone import is_aware
+
+from sentry import nodestore
+from sentry.types.issues import GroupType
+
+
+@dataclass(frozen=True)
+class IssueEvidence:
+    name: str
+    value: str
+    # Whether to prioritise displaying this evidence to users over other issue evidence. Should
+    # only be one important row per occurrence.
+    important: bool
```

Comment:
What do you think about requiring that at least one `important` be `True`? I'm thinking about how I'll use this later on and it'd be kind of annoying to have to check for it, and then either randomly pick one or show nothing if none are important.

Reply:
i like allowing none and showing nothing. should be easier to build up a base issue category and then add the boolean once the team/developer is more familiar with the format instead of adding extra work upfront

Reply:
We could do this, or we could just arbitrarily choose one piece of evidence to be important if nothing is provided.

I guess related to this - should we require that people pass at least once piece of evidence?

Reply:
I am biased towards making it as easy as possible to get set up for new category types etc. As long as a lack of evidence doesn't blow anything up or is too much effort to code around I feel like we don't need to require it.

Plus developers/teams will see the lack of evidence on the other end with the integration. Once that's going that will ideally lead to them then adding the evidence with a better understanding of the system

Reply:
I'm also fine with displaying a random one, but (as unlikely as it is) you're also ratcheting up the severity if a bug ships that causes an omission of the `required` evidence. I just like the system being a bit more fault tolerant

Reply:
Yeah, I'm fine with not requiring it at all. We can also provide a property on the `IssueOccurrence` to return the important piece of info that returns `Optional[Evidence]`, so that the logic of searching through the evidence is contained. Might add that now

Reply:
If we don't require evidence would it be an empty array or None?

Reply:
I think an empty array makes sense